app/Http/Filters/UserFilter.php

```diff
@@ -10,7 +10,13 @@ class UserFilter extends BaseFilter
 {
     public function role(int $role_id)
     {
-        $this->builder->where('role_id', $role_id);
+        if(Role::find($role_id)->name == 'Direct Sales Agent'){
```

Comment:
What is going on here?

Reply:
if searching for dsa, include role 44 and 45, if not, just do normal search